pontoon/base/templates/persona-migration/login.html

```diff
@@ -0,0 +1,30 @@
+{% extends 'socialaccount/base.html' %}
+
+{% load socialaccount %}
+
+{% block extend_js %}
+  {% providers_media_js %}
+{% endblock extend_js %}
+
+{% block content %}
+    {% if user.is_authenticated %}
+      {% if not user.has_fxa %}
+      <p>You have to connect your persona account with your current firefox account</p>
+      <ul>
+        <li><a href="{% provider_login_url "fxa" process="connect" %}">Firefox accounts</a></li>
+      </ul>
+      {% else %}
+
+      <p>You have to connect your firefox account with your current persona account</p>
+      <ul>
+        <li><a href="{% provider_login_url "persona" process="connect" %}">Persona</a></li>
+      </ul>
+      {% endif %}
+    {% else %}
```

Comment:
We can skip the `else` block if you add the `@login_required` decorator to https://github.com/mozilla/pontoon/pull/436/files#diff-717e3b720db25a1f7e95df8c9c361fd2R124.